### PR TITLE
Document (Omni/Spot)Light3D ignoring Node3D's `scale` property

### LINKED ISSUE #67058: Lights do not scale
### Godot version

Godot 4 beta 2

### System information

Windows 10, Vulkan

### Issue description

The radius and size of omni and spot lights do not scale.

This limits the reusability of sub-scenes with lights in them to scenarios when they are not scaled, otherwise the lighting will change in unexpected ways.

### Steps to reproduce

Create a sub-scene. Place some meshes and an omni or spot light into it.
Instantiate the sub-scene in a top level scene multiple times and scale them differently:
LightsDoNotScale

### Minimal reproduction project

[LightsDoNotScale.zip](https://github.com/godotengine/godot/files/9737974/LightsDoNotScale.zip)

### PR DESCRIPTION
**Pull request by Calinou**
`master` version of https://github.com/godotengine/godot/pull/67099.

This closes https://github.com/godotengine/godot/issues/67058.

### CODE REVIEW

File: doc/classes/Light3D.xml [hunk 1]
```diff
@@ -55,6 +55,7 @@
 		</member>
 		<member name="light_angular_distance" type="float" setter="set_param" getter="get_param" default="0.0">
 			The light's angular size in degrees. Increasing this will make shadows softer at greater distances. Only available for [DirectionalLight3D]s. For reference, the Sun from the Earth is approximately [code]0.5[/code].
+			[b]Note:[/b] [member light_angular_distance] is not affected by [member Node3D.scale] (on the light itself or its parents).
```
Comment (skyace65) [inline doc/classes/Light3D.xml:58]:
```suggestion
			[b]Note:[/b] [member light_angular_distance] is not affected by [member Node3D.scale] (The light's scale or its parent's scale).
```

**Review by skyace65 — APPROVED** [review]
Same thing as my review for the 3.x version of this. I think the bit in the parenthesis could be more clear, but I'll defer to other peoples opinion on this. The rest of this looks good though.

File: doc/classes/Light3D.xml (hunk 1 above)
Comment (clayjohn) [reply doc/classes/Light3D.xml:58]: I also think skyace's suggestion is slightly better (except "The" should have a lowercase "t"

**Review by clayjohn — APPROVED** [review]

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Wow, you did a whole bunch of docs PRs :) Thanks, appreciated! 🎉

**Pull request merged**